packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -1,71 +1,38 @@
-    import { appRouter, ledgerRouter } from '@core/router'
-    import { LedgerAppName } from 'shared/lib/typings/ledger'
-    import { formatUnitBestMatch } from 'shared/lib/units'
-    import { getProfileDataPath, walletSetupType } from 'shared/lib/wallet'
-    import { AvailableExchangeRates, CurrencyTypes } from 'shared/lib/typings/currency'
-    import { Locale } from '@core/i18n'
+    import { LOG_FILE_NAME, migration, migrationLog, resetMigrationState } from 'shared/lib/migration'
+    import { activeProfile, updateProfile } from 'shared/lib/profile'
+    import { AppRoute, appRouter, ledgerRouter } from '@core/router'
+    import { api, getProfileDataPath, walletSetupType } from 'shared/lib/wallet'
+    import { Locale, localize } from '@core/i18n'
     import { SetupType } from 'shared/lib/typings/setup'
+    import { showAppNotification } from '@lib/notifications'
+    import { openPopup } from '@lib/popup'
+    import { getDefaultStrongholdName } from '@lib/utils'
 
     export let locale: Locale
 
     const { didComplete } = $migration
-
     const wasMigrated = $didComplete
 
-    let localizedBody = 'body'
-    let localizedValues = {}
     let logExported = false
+    let isNotALedgerProfile: boolean = false
+    let exportStrongholdBusy = false
+    let exportStrongholdMessage = ''
 
     onMount(() => {
         if (!wasMigrated) {
-            if ($walletSetupType === SetupType.FireflyLedger) {
-                localizedBody = 'fireflyLedgerBody'
-            }
-            // This is the last screen in onboarding for all flows i.e., if you create a new wallet or import stronghold
-            // When this component mounts, ensure that the profile is persisted in the local storage.
-            $newProfile = { ...$newProfile, name: $newProfile.id }
-            saveProfile($newProfile)
-            setActiveProfile($newProfile.id)
-
-            newProfile.set(null)
-        } else {
-            if ($walletSetupType === SetupType.TrinityLedger) {
-                localizedBody = 'trinityLedgerBody'
-                localizedValues = { legacy: LedgerAppName.IOTALegacy }
-
-                updateProfile('ledgerMigrationCount', $activeProfile.ledgerMigrationCount + 1)
+            if ($walletSetupType === SetupType.FireflyLedger || $walletSetupType === SetupType.TrinityLedger) {
+                isNotALedgerProfile = false
             } else {
-                localizedBody = 'softwareMigratedBody'
+                isNotALedgerProfile = true
             }
         }
     })
 
-    const fiatbalance = formatCurrency(
-        convertToFiat(
-            // Only show actually migrated balance to user
-            $totalMigratedBalance,
-            get(currencies)?.[CurrencyTypes.USD],
-            get(exchangeRates)?.[AvailableExchangeRates.USD]
-        ),
-        AvailableExchangeRates.USD
-    )
-
     const handleContinueClick = (): void => {
```

Comment:
I think it makes no sense to continue to another view after this view, unless the user wants to migrate another profile